src/test/java/com/openshift/internal/restclient/DefaultClientTest.java

```diff
@@ -97,6 +98,8 @@ private DefaultClient givenClient(URL baseUrl, String token, String user) {
 	
 	@SuppressWarnings("serial")
 	@Test
+
+	@Ignore //filtering is done serverside
```

Comment:
I would remove these tests all together since we have an integration test that supports.  Integration will be enabled on a regular basis hopefully after the first of the year